<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editHabit</class>
 <widget class="QWidget" name="editHabit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(243, 243, 243);
font: 75 12pt &quot;Century Gothic&quot;;

</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="descriptionIn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Arial&quot;;

</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="minimizeB">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/minimize.png</normaloff>:/res/minimize.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/editHabitText.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>381</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(243, 243, 243);
font: 75 12pt &quot;Century Gothic&quot;;

</string>
      </property>
      <property name="text">
       <string>Name *</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameIn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Arial&quot;;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="closeB">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/close.png</normaloff>:/res/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="applyB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(71, 71, 71);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/habitApply.png</normaloff>:/res/habitApply.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>155</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(243, 243, 243);
font: 75 12pt &quot;Century Gothic&quot;;


</string>
      </property>
      <property name="text">
       <string>Grace (Days)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="graceSpinner">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Arial&quot;;

</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="window">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/addHabitWindow.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/version.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelB">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);
color: rgb(71, 71, 71);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/cancelHabitButton.png</normaloff>:/res/cancelHabitButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(243, 243, 243);
font: 75 12pt &quot;Century Gothic&quot;;

</string>
      </property>
      <property name="text">
       <string>Question *</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="questionIn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Arial&quot;;

</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>window</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>minimizeB</zorder>
  <zorder>label_6</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>closeB</zorder>
  <zorder>applyB</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>label_4</zorder>
  <zorder>cancelB</zorder>
  <zorder>layoutWidget_4</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
